docs/en/1_exist_data_model.md

```diff
@@ -580,7 +580,7 @@ This tool accepts several optional arguments, including:
 - `--no-validate` (**not suggested**): Disable evaluation during training.
 - `--work-dir ${WORK_DIR}`: Override the working directory.
 - `--resume-from ${CHECKPOINT_FILE}`: Resume from a previous checkpoint file.
-- `--options 'Key=value'`: Overrides other settings in the used config.
+- `--cfg-options load_from=${CHECKPOINT_FILE}`: Load the model weights from a previous checkpoint file.
```

Comment:
should also fix line 587, meanwhile, originally line 583 means cfg-options, now it only describes load_from, we should update them like
```
- `--options 'Key=value'`: Overrides other settings in the used config. For example, you can use `--cfg-options load_from=${CHECKPOINT_FILE}` to load the model weights from a previous checkpoint file.
```